src/MessagingInput/MessagingInput.less

```diff
@@ -110,8 +112,10 @@
   .border--m(@neutral_silver);
 
   .TextArea--input {
+    .margin--y--none();
+    .padding--y--xs();
+    padding-right: 2.6rem;
```

Comment:
Was a specific choice or trial and error to have the intended design?

Reply:
this was trial and error - it was a delicate balance between having enough spacing for the scroll bar and altering the spacing as little as possible.